config/settings/test.yml

```diff
@@ -1,6 +1,10 @@
 app_users:
   test-user: 'password'
 
+cors:
+  allow_origin_url: 'https://embed-dev.stanford.edu'
```

Comment:
Any setting that we add should have a default setting in `settings.yml`, and then environmental exceptions should be placed in the environment specific `.yml` file.

I think we could simply move this to `settings.yml` and let that point to `embed.stanford.edu`.  Then if we need to configure it differently we can do so a `.local.yml` file (or on the server in the appropriate environment file.

This would mirror how we configure some of our other public services in this repo.